From b3911b53e5cf5838826e33169a3d2b8dcc6d40c9 Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@freescale.com>
Date: Wed, 12 Dec 2012 12:45:37 -0600
Subject: [PATCH 12/53] ENGR00236827-1: Enable MPR121 capacitive button driver
 in imx5 android config

Enable MPR121 capacitive button driver in imx5 android config.

Signed-off-by: Nitin Garg <nitin.garg@freescale.com>
---
 arch/arm/configs/imx5_android_defconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx5_android_defconfig b/arch/arm/configs/imx5_android_defconfig
index 2974b2b..f74b706 100644
--- a/arch/arm/configs/imx5_android_defconfig
+++ b/arch/arm/configs/imx5_android_defconfig
@@ -1278,6 +1278,7 @@ CONFIG_KEYBOARD_GPIO=y
 # CONFIG_KEYBOARD_LM8323 is not set
 # CONFIG_KEYBOARD_MAX7359 is not set
 # CONFIG_KEYBOARD_MCS is not set
+CONFIG_KEYBOARD_MPR121=y
 # CONFIG_KEYBOARD_IMX is not set
 # CONFIG_KEYBOARD_NEWTON is not set
 # CONFIG_KEYBOARD_OPENCORES is not set
-- 
1.7.10.4

